moshi/src/test/java/com/squareup/moshi/JsonStringQualifierRecipe.java

```diff
@@ -0,0 +1,64 @@
+package com.squareup.moshi;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.reflect.Type;
+import java.util.Set;
+import okio.BufferedSink;
+import okio.BufferedSource;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public final class JsonStringQualifierRecipe {
```

Comment:
I initially wrote this before deciding to move it to the recipe subproject instead. Weirdly though, _this_ version works but the recipe version in Kotlin does not, despite appearing to have identical implementations. The recipe version always omits the leading curly brace 🤔